modules/ocf_irc/manifests/services.pp

```diff
@@ -0,0 +1,36 @@
+class ocf_irc::services {
+  package { 'anope':; }
+
+  service { 'anope':
+    ensure  => 'running',
+    require => [Package['anope'], Service['inspircd']],
+  }
+
+  $passwords = parsejson(file("/opt/puppet/shares/private/${::hostname}/services-passwords"))
+
+  File {
+    require => Package['anope'],
+    notify  => Service['anope'],
+    owner   => 'irc',
+    group   => 'irc',
+  }
+
+  file {
+    '/etc/default/anope':
+      content => 'START=yes',
```

Comment:
Haha, thanks for catching that, I did that for the IRCd default file but forgot it here!